com.unity.render-pipelines.high-definition/Documentation~/HDRP-Features.md

```diff
@@ -233,6 +233,8 @@ To decrease aliasing for the main cascade shadow maps, you can apply different f
 
 For punctual and area light shadows, HDRP allows for dynamic resolution based on how much screen the light covers. HDRP rescales the content of the shadow atlas when it would be otherwise full.  For more information on the filtering algorithms that HDRP uses, see the [filtering qualities documentation](HDRP-Asset.md#filtering-qualities).
 
+HDRP also offers a way to cache shadow maps, allowing to update shadow maps on demand or for dynamic objects only. Using this effectively can greatly improve the performance. More information on the  [Shadows](Shadows-in-HDRP.md#ShadowUpdateMode) documentation page.
```

Comment:
Please double-check I preserved the meaning here: 

HDRP also allows you to cache shadow maps. This means you can update shadow maps when needed or update them for dynamic objects only. This method can improve performance significantly. For more information, see  [Shadows](Shadows-in-HDRP.md#ShadowUpdateMode).

Reply:
It's good, only thing is that I'd keep the "on demand" instead of "when needed" as they are subtly different things and the "on demand" also matches the name on the light component. If that sounds good to you ofc. 

Will change soon! (as soon as I can switch back branches)

Reply:
Revised suggestion: 

HDRP also allows you to cache shadow maps. This means you can update shadow maps when you request an update or update them for dynamic objects only. This method can improve performance significantly. For more information, see Shadows.